app/views/videos/show.html.haml

```diff
@@ -6,8 +6,39 @@
       .video_info.col-sm-3
         %header
           %h3= @video.title
-          %span Rating: 4.5/5.0
+          %span Rating: #{@video.average_rating}/5.0
         %p= @video.description
         .actions
           %a.btn.btn-primary(href="") Watch Now
-          %a.btn.btn-default(href="") + My Queue
\ No newline at end of file
+          %a.btn.btn-default(href="") + My Queue
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for [@video, Review.new] do |f|
+        %fieldset
+          .form-group
+            %label Rate this video
+            .row
+              .col-sm-3
+                = f.select :rating, options_for_select([5,4,3,2,1].map {|rating| [rating.to_s + " Star".pluralize(rating), rating]},["5 Stars", 5])

```

Comment:
Thanks @tomtomecek . 
That was a noob error too. I tried to use pluralize that way in the rails console and didn´t work. I googled it and found the other pluralize and worked. 

Now i see what is the difference. The first one (the one you told me) is  the method in the ActionView::Helpers::TextHelper . That means is a helper and only should work in the views, while the one I was using is the "pluralize" method from the string class in ruby (and that works in the console). 
Yours is better, my fault was trying it in the console and not in the view.